snyk/client.py

```diff
@@ -40,13 +42,18 @@ def __init__(
             "Authorization": "token %s" % self.api_token,
             "User-Agent": user_agent,
         }
-        self.api_post_headers = self.api_headers
+        self.api_post_headers = copy.deepcopy(self.api_headers)
         self.api_post_headers["Content-Type"] = "application/json"
+        self.api_patch_headers = copy.deepcopy(self.api_headers)
```

Comment:
```suggestion
        self.api_patch_headers = dict(self.api_headers)
```
Same here.  Why do we need to duplicate them?

Reply:
The same reason as stated above.